libraries/AP_Math/vector3.cpp

```diff
@@ -251,9 +251,13 @@ void Vector3<T>::rotate(enum Rotation rotation)
         z = -sin_pitch * tmpx + cos_pitch * tmpz;
         return;
     }
-    case ROTATION_CUSTOM: // no-op; caller should perform custom rotations via matrix multiplication
-        return;
+    case ROTATION_MAX:
+    case ROTATION_CUSTOM:
+        // no-op; custom rotations not supported
+        break;
     }
+    // rotation invalid or not supported
+    AP::internalerror().error(AP_InternalError::error_t::flow_of_control);
```

Comment:
What do you think about setting a bit aside for this one - at least for a while?

Reply:
new bit for invalid rotation